cmd/config/aws/create_role_cmd.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/armory/armory-cli/pkg/auth"
+	"github.com/armory/armory-cli/pkg/cmdUtils"
+	cliconfig "github.com/armory/armory-cli/pkg/config"
+	"github.com/armory/armory-cli/pkg/util"
+	"github.com/manifoldco/promptui"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	log "go.uber.org/zap"
+	"io"
+)
+
+const (
+	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
+	configCreateRoleLong  = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n\n" +
+		"For usage documentation, visit TODO"
+	configCreateRoleExample = "armory config aws create-role"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
```

Comment:
should this be different for different Armory environments (e.g., what if I want to try the whole Lambda setup workflow against staging)?

Reply:
nvm, you did do that! ignore me.